src/styles/home.scss

```diff
@@ -281,6 +281,76 @@
   @include primary-cta;
 }
 
+// ---------------------------------------------------------------------------
+// Launch Sequences
+// ---------------------------------------------------------------------------
+
+
+  .launch__content {
+    display: flex;
+    flex-direction: column;
+    text-align: center;
+    margin: 80px 0 40px 0;
+    width: 100%;
+  }
+  
+  
+  .launch__header {
+    @include h4-text;
+    color: $pale-yellow !important;
+  }
+  
+  .launch__countdown{
+    @media (min-width: $sm) {
+      font-size: 72px;
+    }
+    font-size: 40px;
+    margin-top: 0.32em;
+    margin-bottom: 0.04em;
+    line-height: 1.0;
+  }
+  
+  .launch__labels {
+    display: inline-block;
+    flex-basis: flex;
+  }
+
+  li {
+    @media (min-width: $sm) {
+      padding-left: 2.5em;
+      padding-right: 1.6em; 
+      font-size: 20px;
+    }
+    display: inline-block;
+    font-size: 16px;
```

Comment:
Bc our styling is global (which is unfortunate, if we were using scoped css this would be fine), this will apply to all list items in the site. Use a class instead

Reply:
fixed :DD